media/engine/webrtc_voice_engine.cc

```diff
@@ -1854,14 +1854,6 @@ void WebRtcVoiceMediaChannel::SetSend(bool send) {
   // may take time on some platforms, e.g. Android).
   if (send) {
     engine()->ApplyOptions(options_);
```

Comment:
Could you either comment it out or leave a comment saying "we don't do InitRecording() here because ...".  It makes future merges easier.

Reply:
I wrapped it in #if false and left a comment.